server/fishtest/templates/elo_results.mak

```diff
@@ -2,9 +2,14 @@
 
 <%
   def get_run_style(run):
+    ret=''
     if 'style' in run['results_info']:
-      return 'background-color:' + run['results_info']['style']
-    return ''
+      ret+='background-color:' + run['results_info']['style']+';'
+    tc=run['args']['tc']
+    new_tc=run['args'].get('new_tc',tc)
+    if tc!=new_tc:
+       ret+='border-style: solid; border-color: Pink;  border-width: medium; border-radius: 0;'
```

Comment:
- double space after "Pink;"
- for red border, the double semicolon ";;" is right? (in chrome using only one don't change the result)
```
<pre style="background-color:;border-style: solid; border-color: Pink;  border-width: medium; border-radius: 0;;white-space:nowrap;" class="elo-results">
```
- for black border:
  - PROD:
  ```
  <pre style="background-color:;white-space:nowrap;" class="elo-results">
   ```
  - DEV (double semicolon ";;"):
   ```
  <pre style="background-color:;;white-space:nowrap;" class="elo-results">
   ```